cinn/runtime/flags.cc

```diff
@@ -25,6 +28,9 @@ DEFINE_bool(cinn_cudnn_deterministic,
             "true, the algorithm is deterministic.");
 #endif
 
+DEFINE_bool(cinn_use_new_fusion_pass, false, "Whether use the new op_fusion and fusion_merge pass.");
+DEFINE_bool(cinn_sync_run, false, "Whether sync all devices after each instruction run, which is used for debug.");
```

Comment:
```suggestion
using ::GFLAGS_NAMESPACE::BoolFromEnv;

DEFINE_bool(cinn_use_new_fusion_pass, BoolFromEnv("FLAGS_cinn_use_new_fusion_pass", false), "Whether use the new op_fusion and fusion_merge pass.");
DEFINE_bool(cinn_sync_run, BoolFromEnv("FLAGS_cinn_sync_run", false), "Whether sync all devices after each instruction run, which is used for debug.");
```
可以这么定义，这样就不需要下面手写的`ParseFromEnv`、`GetCinnUseNewFusionPassFromEnv`等函数。直接使用`FLAGS_cinn_use_new_fusion_pass`和`FLAGS_cinn_sync_run`即可。

Reply:
done